### PR TITLE
WIP - Upgrading to Bootstrap 4 and refactoring views

### PR DESCRIPTION
**Pull request by mindblender**
# DO NOT MERGE YET

Updating bootstrap to v4 and Font Awesome to v5.

Modifying views to make them native templates and easier to maintain.

### CODE REVIEW

File: webapp/resources/cas-theme-default.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
-standard.custom.css.file=/css/cas.css
-admin.custom.css.file=/css/admin.css
+cas.standard.css.file=/css/cas.css
```
Comment (mmoayyed) [inline webapp/resources/cas-theme-default.properties]: Documentation must be updated with the new settings.

**Comment by mmoayyed** [comment]
Also, running a clean build produces:

```
running with these values:
        sassPath: ./resources/static/sass
        npmPath: ./node_modules
        cssPath: ./resources/static/css
Error in plugin 'sass'
Message:
    resources/static/sass/ssosessions.scss
Error: File to import not found or unreadable: partials/fonts.
       Parent style sheet: /Users/Misagh/cas-server/webapp/resources/static/sass/ssosessions.scs
        on line 3 of resources/static/sass/ssosessions.scss
>> @import "partials/fonts";
   ^

```

File: webapp/resources/cas-theme-default.properties (hunk 1 above)
Comment (mindblender) [reply webapp/resources/cas-theme-default.properties]: Will update the documentation

**Comment by mmoayyed** [comment]
Tested. Looks and works excellent. All hail @mindblender

**Pull request merged**